langchain/src/chains/question_answering/load.ts

```diff
@@ -73,21 +80,23 @@ export const loadQAChain = (
 
 interface StuffQAChainParams {
   prompt?: BasePromptTemplate;
+  verbose?: boolean;
 }
 
 export const loadQAStuffChain = (
   llm: BaseLanguageModel,
   params: StuffQAChainParams = {}
 ) => {
-  const { prompt = QA_PROMPT_SELECTOR.getPrompt(llm) } = params;
-  const llmChain = new LLMChain({ prompt, llm });
+  const { prompt = QA_PROMPT_SELECTOR.getPrompt(llm), verbose } = params;
+  const llmChain = new LLMChain({ prompt, llm, verbose });
```

Comment:
(continue from the previous comment) then pass the `outputKey` to the `LLMChain` constructor. Does this make sense?